homework/style.css

```diff
@@ -1,3 +1,142 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 16px;
+}
+
+body {
+  font-family: 'Roboto', Georgia, serif;
+  line-height: 1.5rem;
+  margin: 0 auto;
+  padding: 0;
+  background-color: #fffbe6;
+  color: #2747d4;
+  max-width: 768px;
+}
 .alert-error {
   color: red;
+  margin: 10px;
+  padding: 20px 0 20px 10px;
+  background:pink;
+}
+
+
+header{
+  background-color: #08176e;
+  color: #fffbe6;
+  border: 5px;
+  padding: 20px;
+  margin: 10px;
+  font-size: 20px;
+  flex-flow: row nowrap;
+  align-items: center;
+}
+
+.Header {
+  color: #fffbe6;
+  font-size: 0.9rem;
+  text-align: center;
+  float: left;
+}
+
+select {
+  position: relative;
+  font-size: 0.8rem;
+  width: 32%;
+  line-height: 1.3;
+  height: 2rem;
+  padding: 3px;
+  max-width: 50%;
+  margin-left: 1rem;
+  cursor: pointer;
+  outline: none;
+}
+
```

Comment:
You get a less jumpy user experience if you fix the left and right side to each take up half of the width:

```css
#left-side,
#right-side {
  flex: 1;
}
```